Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+
+--
+-- TOC entry 214 (class 1259 OID 17338)
+-- Name: PROTEIN; Type: TABLE; Schema: PQP; Owner: postgres
+--
+
+CREATE TABLE "PQP"."PROTEIN" (
+    "PROTEIN_ID" integer NOT NULL,
+    "PROTEIN_ACCESSION" text,
+    "DECOY" integer
+);
+
+
+ALTER TABLE "PQP"."PROTEIN" OWNER TO postgres;
+
+--
+-- TOC entry 218 (class 1259 OID 17370)
+-- Name: TRANSITION; Type: TABLE; Schema: PQP; Owner: postgres
+--
+
+CREATE TABLE "PQP"."TRANSITION" (
+    "TRANSITION_ID" integer NOT NULL,
+    "TRAML_ID" text,
+    "PRODUCT_MZ" text,
+    "CHARGE" text,
+    "TYPE" character varying(5),
+    "ANNOTATION" text,
+    "ORDINAL" integer,
+    "DETECTING" integer,
+    "IDENTIFYING" integer,
+    "QUANTIFYING" integer,
+    "LIBRARY_INTENSITY" real,
+    "DECOY" integer
+);
+
+
+ALTER TABLE "PQP"."TRANSITION" OWNER TO postgres;
+
+--
+-- TOC entry 219 (class 1259 OID 17384)
+-- Name: FeatureXML; Type: TABLE; Schema: XML; Owner: postgres
+--
+
+CREATE TABLE "XML"."FeatureXML" (
+    "FeatureMap" xml NOT NULL,
+    "Sample_Name" text NOT NULL,
+    "Investigation_ID" integer NOT NULL,
+    "Study_ID" integer NOT NULL,
+    "Assay_ID" integer NOT NULL,
+    "ID" integer NOT NULL,
+    "mzTab-ID" text NOT NULL
+);
+
+
+ALTER TABLE "XML"."FeatureXML" OWNER TO postgres;
+
+--
+-- TOC entry 224 (class 1259 OID 18192)
+-- Name: FeatureXML_ID_seq; Type: SEQUENCE; Schema: XML; Owner: postgres
+--
+
+CREATE SEQUENCE "XML"."FeatureXML_ID_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE "XML"."FeatureXML_ID_seq" OWNER TO postgres;
+
+--
+-- TOC entry 2973 (class 0 OID 0)
+-- Dependencies: 224
+-- Name: FeatureXML_ID_seq; Type: SEQUENCE OWNED BY; Schema: XML; Owner: postgres
+--
+
+ALTER SEQUENCE "XML"."FeatureXML_ID_seq" OWNED BY "XML"."FeatureXML"."ID";
+
+
+--
+-- TOC entry 212 (class 1259 OID 17326)
+-- Name: TraML; Type: TABLE; Schema: XML; Owner: postgres
+--
+
+CREATE TABLE "XML"."TraML" (
+    "GeneName" text,
+    "ProteinName" text,
+    "PeptideSequence" text,
+    "FullPeptideName" text,
+    transition_group_id text,
+    transition_name text,
+    "Annotation" text,
+    "PrecursorMz" double precision,
+    "MS1 Res" text,
+    "ProductMz" double precision,
+    "MS2 Res" text,
+    "Dwell" integer,
+    "Fragmentor" integer,
+    "CollisionEnergy" double precision,
+    "CellAcceleratorVoltage" integer,
+    "LibraryIntensity" double precision,
+    decoy integer,
+    "LabelType" text,
+    "PrecursorCharge" integer,
+    "FragmentCharge" integer,
+    "FragmentType" text,
+    "FragmentSeriesNumber" integer,
+    quantifying_transition integer,
+    identifying_transition integer,
+    detecting_transition integer,
+    "Tr_recalibrated" double precision,
+    "RetentionTime" double precision,
+    "PeptideGroupLabel" text,
+    "UniprotId" text,
+    "CompoundName" text,
+    "SMILES" text,
+    "SumFormula" text,
+    "mzTab-ID" text NOT NULL,
+    "ID" integer NOT NULL
```

Comment:
As with the above `ID` columns, we need to be consistent here.  It also looks like this internal ID does not match the `TRAML_ID`s used in the previous tables.